app/controllers/api/teams.php

```diff
@@ -314,9 +318,13 @@
     ->inject('dbForProject')
     ->inject('locale')
     ->inject('mails')
+    ->inject('messaging')
     ->inject('events')
-    ->action(function (string $teamId, string $email, array $roles, string $url, string $name, Response $response, Document $project, Document $user, Database $dbForProject, Locale $locale, Mail $mails, Event $events) {
+    ->action(function (string $teamId, string $userId, string $email, string $phone, array $roles, string $url, string $name, Response $response, Document $project, Document $user, Database $dbForProject, Locale $locale, Mail $mails, EventPhone $messaging, Event $events) {
 
+        if (empty($userId) && empty($email) && empty($phone)) {
```

Comment:
I think we should make this `exactly one`. If you pass all 3, how does it behave?

Or if it's on purpose for creating, we need to make documentation around what is the priority.

Reply:
we have a priority ID > email > phone for this purpose